src/components/App.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react'
+import EmptyEditor from '../components/Editor/EmptyEditor'
+import NoteList from "./NoteList/NoteList";
 
 class App extends React.Component{
     render() {
+        //height:'100vh'
+        //flex-direction:'row'
```

Comment:
Remember to remove comments

Reply:
i leave comments for now for you to see what i also tried etc.